### ISSUE TITLE
question - polygon packing 

### ISSUE THREAD

**Issue by macguffin** [gameofbombs/pixi-heaven#14]
Does pixi-heaven polygon packing (pixi v5) work in canvas mode, with the legacy plugin enabled?

Thanks

**Comment by ivanpopelyshev** [comment]
No, it doesn't. I know how to implement it there though :) Thanks for the idea!

However, I dont know whether its actually needed. Did you see any projects/renderers that support polygon packing in canvas mode?

**Comment by macguffin** [comment]
No but I have just updated to pixiv5 and was planning to also implement pixi-heaven and poly-packing but a requirement of my games is that they have a canvas fall back.
Definitely not urgent but it would be cool to be able to use it if you ever get round to it.
Thanks
:)
